tests/http_client_mock.py

```diff
@@ -248,7 +248,10 @@ def get_last_call(self) -> StripeRequestCall:
         return StripeRequestCall.from_mock_call(self.func.call_args)
 
     def get_all_calls(self) -> List[StripeRequestCall]:
-        return [StripeRequestCall.from_mock_call(call) for call in self.func]
+        return [
+            StripeRequestCall.from_mock_call(call_args)
+            for call_args in self.func.call_args_list
+        ]
```

Comment:
I used the `get_all_calls` and `get_raw_header` methods for debugging, and fixed some bugs in them. But these 2 methods aren't used anywhere.